comps/comps-foreman-client-el10.xml

```diff
@@ -14,7 +14,7 @@
       <packagereq type="default">foreman_ygg_worker</packagereq>
```

Comment:
Should we drop this too just so we are clear in what we publish and don't publish?